src/main/java/medicaldoctor/controllers/ControllerManager.java

```diff
@@ -0,0 +1,54 @@
+package medicaldoctor.controllers;
+
+import java.io.IOException;
+import java.util.HashMap;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Node;
+import javafx.scene.Parent;
+import javafx.scene.layout.StackPane;
+
+
+public class ControllerManager extends StackPane{
+    private HashMap<String, Node> screen = new HashMap<>();
```

Comment:
can certainly use the fxml name with the extension as the key.

Reply:
I don't think it's necessary, I know it's not elegant, but it works and I already implemented all the controllers like this.

Reply:
maintainability, and easier to fix things now  than later.